realm/realm-library/src/main/java/io/realm/RealmQuery.java

```diff
@@ -404,6 +406,192 @@ public boolean isValid() {
         return this;
     }
 
+    // In
```

Comment:
I think you can actually use `NullTypes` or `NoPrimaryKeyNullTypes` for testing. no need to change the `AllTypes`, then no need to update this binary file. 

Also, you can add test for nullable fields which is missed right now.